inst/gates/Myeloid.RM.tsv

```diff
@@ -0,0 +1,15 @@
+levels	use_as	name
+level1	negative	Erythrocyte.RM
+level1	negative	Epithelial.RM
+level1	negative	Stromal.RM
+level1	negative	AvEp.RM
+level1	positive	pDC.RM
+level1	positive	Bcell.RM
+level1	positive	Myeloid.RM
+level1	positive	Tcell
+level1	positive	NK.RM
+level2	negative	Tcell
```

Comment:
Is there a reason this does Level 1 positive for Tcell and then negative?

Reply:
also, why's it do level1 positive for Bcell?

Reply:
does it need to be redundant with level1 positive for pDC+myeloid.rm and repeat on level2?

Reply:
The gating structure is somewhat modeled after the default gates. Level 1 defines the overall population of cells we are interested in (which is why all of these cell types are positive at level 1) and level 2 segments these cells into their components. We could probably get away with collapsing to one level (which I did for some of the small populations that we are not interested in analyzing further, like epithelial).

Reply:
So I can buy the reasoning for redundantly having positive Bcell on levels 1 and 2, which is like this: https://github.com/carmonalab/scGate_models/blob/master/human/generic/Bcell_scGate_Model.tsv

But I question why we want a level1 positive for Tcell, NK cell, etc. What's the point on this and it seems like it could only be bad? If we were attempting to make some sort of single-gate that handled every population, perhaps that could make sense. but this isnt what's happening here.